tests/app/test_ipexing.py

```diff
@@ -341,24 +363,32 @@ def test_ipex_grant(helpers, mockHelpingNowIso8601, seeder):
                             'title': '400 Bad Request'}
 
 
-def test_multisig_grant_admit(seeder, helpers):
+def test_multisig(seeder, helpers):
```

Comment:
This test is now even longer, so not sure. But the setup of multi-sigs and credentials being issued is expensive so decided to keep it all to one test if that's OK.